src/rules/enforceHeaderRule.ts

```diff
@@ -0,0 +1,61 @@
+import * as ts from "typescript";
+
+import * as Lint from "../lint";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "enforce-header",
```

Comment:
can we call this rule "file-header" instead? "enforce" is a little redundant because all our rules enforce _something_.